packages/types/src/session.ts

```diff
@@ -6,7 +6,7 @@ export interface RequestSession {
 
 export interface Session {
   sid: string;
-  did?: string;
+  did?: string | number;
```

Comment:
https://develop.sentry.dev/sdk/sessions/#session-update-payload

`did` is typed as string in Relay. If we wanna allow number here we have to stringify it before sending.

Reply:
We can change what we allow here but we should only send the stringified version. Which we already do - I think Daniel found that out.